x/evm/embeds/contracts/IOracle.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.19;
+
+/// @dev Oracle interface for querying exchange rates
+interface IOracle {
+  /// @dev queryExchangeRate queries the exchange rate for a given pair
+  /// @param pair the pair to query
```

Comment:
```suggestion
/// @notice Oracle interface for querying exchange rates
interface IOracle {
  /// @notice Queries the exchange rate for a given pair
  /// @param pair The asset pair to query. For example, "ubtc:uusd" is the 
  /// USD price of BTC and "unibi:uusd" is the USD price of NIBI.
  /// @return The exchange rate (a decimal value) as a string.
  /// @dev This function is view-only and does not modify state.
```

I based this on the guidelines [written here](https://github.com/NibiruChain/nibiru/tree/main/x/evm/embeds#precompile-solidity-documentation).